packages/vite/src/node/server/middlewares/static.ts

```diff
@@ -1,12 +1,27 @@
 import os from 'os'
 import path from 'path'
+import { ServerResponse } from 'http'
 import sirv from 'sirv'
 import { Connect } from 'types/connect'
 import { ResolvedConfig } from '../..'
 import { FS_PREFIX } from '../../constants'
 import { cleanUrl, isImportRequest } from '../../utils'
 
-const sirvOptions = { dev: true, etag: true, extensions: [] }
+const sirvOptions = {
```

Comment:
You can `import type { Options } from 'sirv';` and then

```suggestion
const sirvOptions: Options = {
```

This means you don't have to import `ServerResponse` or type the `setHeaders` method.

Reply:
Thanks for the suggestion! I updated this PR now.